src/pages/Feed/Feed.js

```diff
@@ -316,7 +316,7 @@ class Feed extends Component {
             updatedPosts[postIndex] = post
           } else {
             updatedTotalPosts++
-            if (prevState.posts.length >= 2) {
+            if (prevState.posts.length >= 3) {
```

Comment:
`prevState.posts.length >= 3` Currently backend has been set to display post 3 posts per page, Depend on Backend